docs/Information_nodes.rst

```diff
@@ -21,37 +21,31 @@ Node profile description
 | :h:`<h5 style="margin: 5px 0 0 0;">DGX Systems</h5>`                                                                                                            |
 +---------------------------------------+----------+---+------+---------+--------------+--------------+-------------+--------------+--------+---------------------+
-| mila01                                | V100     | 8 |  80  |    2    |      20      |       2      |     512     |      7       | x86_64 |      volta,16gb     |
+| **cn-d[001-002]**                     | A100     | 8 |  128 |    2    |      64      |       1      |    1024     |     14       | x86_64 | ampere,nvlink,40gb  |
 +---------------------------------------+----------+---+------+---------+--------------+--------------+-------------+--------------+--------+---------------------+
-| mila[02-03]                           | V100     | 8 |  80  |    2    |      20      |       2      |     512     |      7       | x86_64 |      volta,32gb     |
+| **cn-e001**                           | V100     | 8 |  40  |    2    |      20      |       1      |     512     |      7       | x86_64 |      volta,16gb     |
 +---------------------------------------+----------+---+------+---------+--------------+--------------+-------------+--------------+--------+---------------------+
-| **cn-d[001-002]**                     | A100     | 8 |  128 |    2    |      64      |       2      |    1024     |     14       | x86_64 | ampere,nvlink,40gb  |
+| **cn-e[002-003]**                     | V100     | 8 |  40  |    2    |      20      |       1      |     512     |      7       | x86_64 |      volta,32gb     |
 +---------------------------------------+----------+---+------+---------+--------------+--------------+-------------+--------------+--------+---------------------+
 | :h:`<h5 style="margin: 5px 0 0 0;">Legacy GPU</h5>`                                                                                                             |
 +---------------------------------------+----------+---+------+---------+--------------+--------------+-------------+--------------+--------+---------------------+
-| kepler3                               | k80      | 8 |  16  |    2    |       4      |       2      |     256     |      3.6     | x86_64 |     kepler,12gb     |
-+---------------------------------------+----------+---+------+---------+--------------+--------------+-------------+--------------+--------+---------------------+
-| kepler4                               | m40      | 4 |  16  |    2    |       4      |       2      |     256     |      3.6     | x86_64 |    maxwell,24gb     |
-+---------------------------------------+----------+---+------+---------+--------------+--------------+-------------+--------------+--------+---------------------+
 | kepler5                               | V100     | 2 |  16  |    2    |       4      |       2      |     256     |      3.6     | x86_64 |      volta,16gb     |
```

Comment:
Labeling a v100 in a Legacy GPU section might bring confusion as to why other v100 are not also legacy? Maybe Legacy Board/Machine could be a better name now?

Reply:
Note the hostname is `kepler` while there is no K80 available, only V100... This is a legacy system. Should I update the description for "Legacy GPU compute nodes"?

Reply:
Yes at least for me that sounds like a better description